doc: Add Akenza Core webhook integration guide (#2787)
TheThingsNetwork/lorawan-stack: doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md

---
title: "Akenza Core Setup"
description: ""
weight: 1
---

This section shows how to prepare Akenza Core setup before creating a Webhook integration on {{% tts %}}.

<!--more-->

> By logging in to your Akenza Core user account and navigating to **QUICK START** tab in the **Environment**, you can see the order of steps to be taken to connect a device and manipulate the data coming from it. 

Reviewer: ```suggestion
Log in to Akenza Core and navigate to the **Quick Start** tab in the **Environment** to see the steps to be taken to connect a device and manipulate the data coming from it. 
```

{{< figure src="quick-start.png" alt="Steps to connect a device" >}}

> Set up a domain by selecting the **Add Domain** button in **Domains** submenu. 

Reviewer: ```suggestion
Set up a domain by selecting the **Add Domain** button in the **Domains** submenu. 
```

> First step is setting up a domain, which can be done by selecting the **ADD DOMAIN** button in **Domains** submenu. 

Reviewer: ```suggestion
Set up a Domain by selecting the **Add Domain** button in **Domains** submenu. 
```

> Give a **Name** to your domain and choose **HTTP** for **Technology** in order to implement a Webhook integration. The **Domain Secret** is auto-generated, while **Uplink Function** and **Downlink Function** may retain a **Passthrough** value. 

Reviewer: ```suggestion
Give a **Name** to your domain and choose **HTTP** for **Technology** to create a Webhook integration. The **Domain Secret** is auto-generated, while **Uplink Function** and **Downlink Function** may retain a **Passthrough** value. 
```

> Give a **Name** to your domain and choose **HTTP** for **Technology** in order to implement a Webhook integration. **Domain Secret** is auto-generated, while **Uplink Function** and **Downlink Function** may retain **Passthrough** value. 

Reviewer: ```suggestion
Give a **Name** to your domain and choose **HTTP** for **Technology** in order to implement a Webhook integration. The **Domain Secret** is auto-generated, while **Uplink Function** and **Downlink Function** may retain a **Passthrough** value. 
```

> Select **SAVE** to finish.

Reviewer: ```suggestion
Select **Save** to finish.
```

{{< figure src="creating-domain.png" alt="Creating a new domain" >}}

> In a similar manner, you need to create a new device type in **Device Types** submenu, where you just need to select **ADD DEVICE TYPE** button, provide a **Name** and click **SAVE**.

Reviewer: ```suggestion
Next, create a new device type in the **Device Types** submenu. Select the **Add Device Type** button, provide a **Name** and click **Save**.
```

{{< figure src="creating-device-type.png" alt="Creating a device type" >}}

> When you have defined a domain and a device type, you can create a new device by selecting **Add Device** button in the **Inventory** submenu. 

Reviewer 2: ```suggestion
After defining the domain and a device type, you can create a new device by selecting **Add Device** button in the **Inventory** submenu. 
```

> When you have defined a domain and a device type, you can create a new device by selecting **ADD DEVICE** button in **Inventory** submenu. 

Reviewer: ```suggestion
When you have defined a domain and a device type, you can create a new device by selecting the **Add device** button in the **Inventory** submenu. 
```

> Provide a **Name** for your device, select previously created device type and domain, and do not forget to generate **Device ID**.

Reviewer: ```suggestion
Provide a **Name** for your device, select the previously created device type and domain, and do not forget to generate a **Device ID**.
```

{{< figure src="creating-device.png" alt="Creating a new device" >}}

> Once the device is created, select it in **Inventory** and scroll down to the bottom of the page. Under **Endpoints** you can find and copy **HTTP Uplink URL** that you need for the Webhook integration.

Reviewer: ```suggestion
Once the device is created, select it in **Inventory** and scroll down to the bottom of the page. Under **Endpoints**, you will find the **HTTP Uplink URL** that you need for the Webhook integration.
```

<!--more-->

> Log in to Akenza Core and navigate to **Quick Start** tab in the **Environment** to see the steps to be taken to connect a device and manipulate the data coming from it. 

Reviewer: ```suggestion
Log in to Akenza Core and navigate to the **Quick Start** tab in the **Environment** to see the steps to be taken to connect a device and manipulate the data coming from it. 
```
